docs/how-to/3rd-party/tensorflow-install.rst

```diff
@@ -72,21 +54,51 @@ Follow these steps:
 Using a wheels package
 -------------------------------------------------------------------------------
 
-To install TensorFlow using the wheels package, use the following command.
+#. Ensure your environment is using a supported Python version for the desired ``tensorflow-rocm`` wheel.
 
-.. code-block:: shell
+   a. To check compatible Python versions, navigate to the desired ROCm release version
+      in `<https://repo.radeon.com/rocm/manylinux/>`__.
+
+   b. In the versioned ROCm release directory, take note of the Python versions associated with the desired
+      wheel version indicated by ``cp`` in the wheels' file names.
+      
+      For example, ``tensorflow_rocm-2.15.0-cp310-cp310-manylinux2014_x86_64.whl`` targets Python 3.10, as indicated by
+      ``cp310``.
 
-   pip install --user tensorflow-rocm==[wheel-version] -f [repo] --upgrade
```

Comment:
I think we need to keep this line as a more general install instruction.  For example, if they need to install for ROCm 6.0.0, or 6.0.2 (any version older than 6.1) then they may need to specify a different repo (not repo.radeon.com)?